applicationinsights-android/src/main/java/com/microsoft/applicationinsights/library/LifeCycleTracking.java

```diff
@@ -149,7 +149,7 @@ private static void unregisterActivityLifecycleCallbacks(Application application
      */
```

Comment:
I think we also need to check this in `registerActivityLifecycleCallbacks ` and `unregisterActivityLifecycleCallbacks ` https://github.com/Microsoft/ApplicationInsights-Android/blob/feature/fix-npe-in-lifecycle-tracking/applicationinsights-android/src/main/java/com/microsoft/applicationinsights/library/LifeCycleTracking.java#L123-L143